### ISSUE TITLE
Plugin error on Atom start

### ISSUE THREAD

**Issue by Spy-Seth** [AtomLinter/linter-eslint#146]
Since the last update, I've this error:

```
SyntaxError: Unexpected end of input
  at Object.parse (native)
  at Object.<anonymous> (/Users/spy-seth/.atom/packages/linter-eslint/lib/linter-eslint.js:120:28)
  at GeneratorFunctionPrototype.next (native)
  at step (/Users/spy-seth/.atom/packages/linter-eslint/lib/linter-eslint.js:7:273)
```

**Comment by astephenb** [comment]
:+1: I have been experiencing this as well.

**Comment by astephenb** [comment]
After some investigation, I found that I didn't have `babel-eslint` installed locally, but was trying to use that as a parse in my eslintrc. The error message was not helpful though.

**Comment by djindjic** [comment]
I have same error after updating atom to 1.0.2 and linter-eslint to 2.2.0. Also I have this warning message:
```bash
[Linter-ESLint] Local/Global ESLint binary not found. Fallback to packaged one.
Set binary in preferences.
```
I have regulary updated global and local eslint 0.24.0

**Comment by iam4x** [comment]
* Is ESLint installed localy or globaly?
* How do you open Atom editor?
* What is the absolute path of Atom when you open it (example with cli `$ atom /path/to/project`)?
* What is the absolute path of the file you are editing while getting this issue?

**Comment by djindjic** [comment]
- in my case, eslint is instaled localy and globaly
- `atom .`
- not sure I understand
- every file in project

It;s ok now, I don't have this issue after updating packages, but in this new version I can see eslint error messages, just underlined syntax errors.

**Comment by iam4x** [comment]
@djindjic What is not working? You don't have the error messages?

**Comment by djindjic** [comment]
There is no more atom errors log, and eslint works fine, but I dont have warning and error mesages if I broke some js syntax.
For example I have `5:7  error  i is defined but never used  no-unused-vars` in command line tool, but in atom there is just underlined line 5 without message. I was functional before latest updates.

**Comment by polarathene** [comment]
Just updated linter and linter-eslint. Previously I only had a similar error(now a warning) about not being able to find eslint. linter-luacheck is disabled not sure why that is erroring, might be related to linter update, possibly the cause of popping up the console on the side too judging by the logged errors.

Win 8.1 x64, running Atom via pinned task bar button. Have run apm uninstall/install on linter-eslint. I do not have a local copy of node installed on the machine at present, linter-eslint can't seem to find/use the version it's meant to have packaged with it? 

![atom errors](http://i.imgur.com/zVXTO2j.png)

**Comment by guillaume86** [comment]
@polarathene I have a local node install and the same error messages (Windows 8.1). Also launching Atom from the pinned taskbar icon (which must be very common on Windows).

**Comment by iam4x** [comment]
This issue is related to #149 let's keep the discussion in one place.
